numpy/lib/tests/test_histograms.py

```diff
@@ -397,6 +398,16 @@ def test_histogram_bin_edges(self):
         edges = histogram_bin_edges(arr, bins='auto', range=(0, 1))
         assert_array_equal(edges, e)
 
+    @requires_memory(free_bytes=100000000 * 3)
```

Comment:
Thanks.  Lets refine this a little bit.  If you check `sample.nbytes`, you will see that it is 8 times larger.   Also the result array (with the current choices) has a significant size.  You could just reduce that size by a lot, even then it probably makes sense to round up a little bit.
(This is a sanity check that we should be good after all.  The decorator also catches `MemoryError` so we should be fine, but I would just round up generously.)

Reply:
Sure. That makes sense. I made it 100 times larger

Reply:
OK, if I do the math, this should be about double of what is needed, and 10 GiB are not forbidding these days, so this should be fine.